src/Psalm/Internal/Provider/ReturnTypeProvider/FilterVarReturnTypeProvider.php

```diff
@@ -55,8 +55,6 @@ public static function getFunctionReturnType(FunctionReturnTypeProviderEvent $ev
         ) {
             $filter_type_type = $second_arg_type->getSingleIntLiteral();
 
-            $filter_type = null;
```

Comment:
I would actually either leave it here or add a default case to the switch below.

Reply:
Did you see this line a little above? https://github.com/vimeo/psalm/pull/7233/files/0b3e1299decdd75fe6cf38acd88391cfea323516#diff-76c8a38ce19c782b153d6a5cbededfe163697de0e37f2f40559f3f07f8eaf318L50

It was just initialized twice

Reply:
Ok then.